inlong-sort/sort-connectors/base/src/main/java/org/apache/inlong/sort/base/dirty/DirtySinkHelper.java

```diff
@@ -136,6 +185,51 @@ public void invokeMultiple(T dirtyData, DirtyType dirtyType, Throwable e,
         }
     }
 
+    private String regexReplace(String pattern, DirtyType dirtyType,
+            String dirtyMessage, List<String> actualIdentifier) throws IOException {
+
+        if (pattern == null) {
+            return null;
+        }
+
+        final Pattern REGEX_PATTERN = Pattern.compile("\\$\\{\\s*([\\w.-]+)\\s*}", Pattern.CASE_INSENSITIVE);
```

Comment:
Suggest changing this regex pattern to a final static variable, or changing the `regexReplace` method to static, which can improve the performance.